### PR TITLE
Added Group 5 semantic tests

### PR DESCRIPTION
**Pull request by minijavacc**

### CODE REVIEW

File: tests/semantic/InvalidMethodCalls.invalid.mj [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class InvalidMethodCalls {
+
+	
+	public static void main(String[] args) {
+		
+		InvalidMethodCalls calls=new InvalidMethodCalls();
+		calls.nothing(true);
+		calls.oneint(100,50);
+		calls.threeints(12, false);
+	}
```
Comment (parttimenerd) [inline tests/semantic/InvalidMethodCalls.invalid.mj]: Please split this test case into three (for each of the three invalid method calls).

File: tests/semantic/MultipleReturns.valid.mj [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	public static void main(String[] args) {
+		MultipleReturns multipleReturns=new MultipleReturns();
+		int c=multipleReturns.complexFunction(5, false, 12, true);
+		int b=multipleReturns.complexFunction(423423, true, -554, false&true);
+		multipleReturns.complexFunction(c, c<b, b*c, c+b*12==12312);
+	}
+
+	public int complexFunction(int a, boolean b, int c, boolean d) {
+		if (a > c && b != d) {
+			if (b & d || a >> 10 <= 15) {
```
Comment (parttimenerd) [inline tests/semantic/MultipleReturns.valid.mj]: Is this really valid MiniJava (I thought that the operators `&` and `>>` aren't supported)?

File: tests/semantic/NestedScopes.valid.mj [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+					 b=b*5;
+					 {
+						 if(b>a)
+						 {
+							 b=a;
+						 }
+					 }
+				 }
+			 }
+			 a++;
```
Comment (parttimenerd) [inline tests/semantic/NestedScopes.valid.mj]: The `++` operator shouldn't be supported either.

File: tests/semantic/ThisTest.valid.mj [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	}
+	
+	public void thisTester(int t)
+	{
+		if(t==t&&this.t==this.t)
+		{
+			this.t=t*2;
+			if(t==this.t)
+			{
+				//error
```
Comment (parttimenerd) [inline tests/semantic/ThisTest.valid.mj]: This kind of comments isn't supported in MiniJava.

**Comment by Rolleander** [comment]
Hab die Punkte gefixed, sollte jetzt stimmen.

**Pull request merged**